### DISCUSSION TITLE
İşlem sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#240] [Zeka Sorusu] [answered]
1, 2, 25, 50, 75 ve 100 sayılarını birer kez kullanarak 383 sayısını elde edin. (Dört işlem kullanarak)

**Comment by AcelyaP** [comment, marked as answer]
50 + 2= 52
52/ 25 = 2.08
2.08 + 1 = 3.08
3.08 x 100 = 308
308 + 75 = 383
